manifests/install.pp

```diff
@@ -75,13 +75,20 @@
     $_package_options = {}
   }
 
+  if $::osfamily =~ /windows/ {
+    $local_package_file_path = windows_native_path("${::puppet_agent::params::local_packages_dir}/${package_file_name}")
+  } else {
+    $local_package_file_path = "${::puppet_agent::params::local_packages_dir}/${package_file_name}"
```

Comment:
Hm, apparently not. It seems like it should be for the platforms w/o package managers.